contrib/debian/rules

```diff
@@ -27,9 +27,7 @@ PRIVATE_LIBS = /usr/lib/aziot-identity-service
 
 override_dh_auto_build:
 	# Default behavior is `make -j1`, which is not quite right for us.
-	make -j \
-		RELEASE=1 \
-		V=1
```

Comment:
These lines were only removed originally as part of a workaround to perform the build outside of the dpkg-buildpackage step.  When that workaround was subsequently removed these probably should have been added back.

Reply:
Reverted.